examples/httpserver_cpu_information.py

```diff
@@ -15,14 +15,15 @@
 from adafruit_httpserver.server import HTTPServer
 
 
-ssid, password = secrets.WIFI_SSID, secrets.WIFI_PASSWORD  # pylint: disable=no-member
+ssid = os.environ.get("WIFI_SSID")
+password = os.environ.get("WIFI_PASSWORD")
```

Comment:
I tested this out on a Feather S2 TFT and found that I had this error raised:
```
  File "code.py", line 18, in <module>
AttributeError: 'module' object has no attribute 'environ'
```

I think maybe we don't have the `environ` api and instead may need to use getenv like: `os.getenv('WIFI_SSID')` I tried changing the code in one of the examples to use getenv instead and I was able to execute it successfully

Reply:
Good catch. Is `os.getenv('WIFI_SSID')` the standard for examples now for wifi connect _not_ via web workflow?

Reply:
> Good catch. Is `os.getenv('WIFI_SSID')` the standard for examples now for wifi connect _not_ via web workflow?

Seems that it is, https://docs.circuitpython.org/en/latest/docs/environment.html

My bad for using the `os.environ`, I wrongly assumed that it will work the same way as in Python, already fixed in teh latest commit. Thanks for testing @FoamyGuy